locale/en.json

```diff
@@ -285,6 +286,318 @@
+            "In some circumstances we may share your data for example, to prevent or detect crime, or to produce anonymised statistics.",
+            "We use Google Analytics to collect data about how a website is used. This anonymous data is shared with Google. Find out about this in our <a class='govuk-link' href='/terms-and-conditions'>terms and conditions</a>."
+          ]
+        },
+        {
+          "title": "Storing and sharing your data internationally",
+          "content": [
+            "Sometimes we need to send your personal information outside of the UK. When we do this we comply with data protection law."
+          ]
+        },
+        {
+          "title": "Your rights",
+          "content": [
+            "You can ask:",
+            "<ul class='govuk-list govuk-list--bullet'><li>to see the personal data that we hold on you</li><li>to have the personal data corrected</li><li>to have the personal data removed or deleted (this will depend on the circumstances, for example if you decide not to continue your claim or application)</li><li>that access to the personal data is restricted (for example, you can ask to have your data stored for longer and not automatically deleted)</li></ul>",
+            "If you want to see the personal data that we hold on you, you can:",
+            "<ul class='govuk-list govuk-list--bullet'><li>complete a form to <a class='govuk-link' href='https://www.gov.uk/government/publications/request-your-personal-data-from-moj'>make a subject access request</a>. Your request goes to the MoJ as data controller.</li><li>write to us: Disclosure Team, Post point 10.38, 102 Petty France, London, SW1H 9AJ</li></ul>",
+            "You can ask for more information about:",
+            "<ul class='govuk-list govuk-list--bullet'><li>agreements we have on sharing information with other organisations</li><li>when we are allowed to pass on personal information without telling you</li><li>our instructions to staff on how to collect, use or delete your personal information</li><li>how we check that the information we hold is accurate and up-to-date</li></ul>",
+            "You can contact the MoJ data protection officer, by:",
+            "<ul class='govuk-list govuk-list--bullet'><li>writing to us: Post point 10.38, 102 Petty France, London, SW1H 9AJ</li><li>emailing: data.compliance@justice.gov.uk</li></ul>"
+          ]
+        },
+        {
+          "title": "How to complain",
+          "content": [
+            "See our <a class='govuk-link' href='https://www.gov.uk/government/organisations/hm-courts-and-tribunals-service/about/complaints-procedure'>complaints procedure</a> if you want to complain about how we've handled your personal data.",
+            "Write to: Post point 10.38, 102 Petty France, London, SW1H 9AJ",
+            "Email: data.compliance@justice.gov.uk",
```

Comment:
Should this have a mailto: ?

Reply:
This is just content the mailto: should be in the template file shouldn't it